letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -1641,18 +1664,9 @@ if __name__ == '__main__':
 
 UNLIKELY_EOF
     # -------------------------------------------------------------------------
-    set +e
-    PEEP_OUT=`"$VENV_BIN/python" "$TEMP_DIR/peep.py" install -r "$TEMP_DIR/letsencrypt-auto-requirements.txt"`
-    PEEP_STATUS=$?
-    set -e
+    InstallRequirements "setuptools-requirements.txt"
```

Comment:
Btw, I realized why you need to break this up into 2 files: peep runs all its invocations of pip in a single process, calling out to `pip.main()` directly. I did that so I wouldn't accidentally invoke some other venv's (or the system's) copy of pip, no matter how screwed up `$PATH` happened to be.